WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModel.cs

```diff
@@ -9,6 +9,7 @@
 using System.Reactive;
 using System.Reactive.Disposables;
 using System.Reactive.Linq;
+using System.Windows.Input;
```

Comment:
Probably it is just a leftover.

Reply:
using System.Windows.Input;

this is for ICommand properties.

Reply:
Yeah but this file is from GUI project